torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -506,6 +507,8 @@ def speculate_subgraph(
     restore_side_effects=True,
     should_flatten_outputs=False,
     under_activation_checkpoint=False,
+    supports_input_mutation=True,
+    supports_aliasing=True,
```

Comment:
btw I would still prefer these two to default to False because that is more strict.

Reply:
I tried to toggle the default but things broke for many HOPS. So tracking it here for now - https://github.com/pytorch/pytorch/issues/149989

Reply:
What I meant was:
- the default values should be False
- to preserve BC, we should change all callsites to use True for these kwargs.
- Then the future work becomes an exercise of going through each HOP one at a time and fixing the flags, and we get the benefit that the default value is False for future HOPs. 

Is it possible to do the first two bullets in this PR or a follow-up? I don't think this should be difficult